core/src/main/java/io/seata/core/rpc/DefaultServerMessageListenerImpl.java

```diff
@@ -147,7 +132,7 @@ public void onCheckMessage(RpcMessage request, ChannelHandlerContext ctx, Server
         try {
             sender.sendResponse(request, ctx.channel(), HeartbeatMessage.PONG);
         } catch (Throwable throwable) {
-            LOGGER.error("", "send response error", throwable);
+            LOGGER.error("send response error", throwable);
```

Comment:
please user three parameter method
`LOGGER.error("send response error:{}",throwable.getMessage(), throwable);`

Reply:
> please user three parameter method
> `LOGGER.error("send response error:{}",throwable.getMessage(), throwable);`

ok, I will fix it.